tools/klee-stats/README.md

```diff
@@ -0,0 +1,130 @@
+#KLEE-Stats
+
+##Info
+klee-stats is a script to help interpret data outputted by klee. It is used with a corresponding config file and heavily uses Python's eval and exec capabilities. The config file follows the Windows INI structure however in the key-value pairs, the values are stored as segments of Python code.
```

Comment:
Thanks, @andreamattavelli , I would actually like to move this README to the website, I think the right place is there.